compiler/resolution/lowerIterators.cpp

```diff
@@ -1246,6 +1246,17 @@ expandIteratorInline(ForLoop* forLoop) {
   Symbol*   ic       = forLoop->iteratorGet()->symbol();
   FnSymbol* iterator = getTheIteratorFn(ic);
 
+  if (fReportInlinedIterators) {
+    ModuleSymbol *mod = toModuleSymbol(iterator->getModule());
+    INT_ASSERT(mod);
+
+    if (developer || mod->modTag == MOD_USER) {
+      printf("Inlined iterator (%s) in module %s (%s:%d)\n", iterator->cname,
```

Comment:
I think the location where the iterator is getting inlined e.g. `forLoop` is as important as the location where the iterator is defined.

Reply:
I don't really disagree, but for now I want to keep the output consistent with `--report-optimized-loop-iterators`.